src/vl-map-delete-action.js

```diff
@@ -37,10 +37,32 @@ export class VlMapDeleteAction extends VlMapLayerAction {
     }
   }
 
+  /**
+   * Zet de functie die wordt uitgevoerd na het uitvoeren van de verwijder actie
+   *
+   * @param {Function} callback functie met volgende argumenten:
+   *                            - {[ol.Feature]} de te verwijderen features
+   *                            - {Function} resolve callback met ol.Feature als argument die verwijderd wordt
+   *                            - {Function} reject callback zonder argument waarbij de highlight verwijderd wordt
+   */
+  onDelete(callback) {
```

Comment:
moet dit geen setter zijn?

Reply:
Ik heb dezelfde syntax genomen als bij onSelect bij de select action. Ik zou het kunnen veranderen, maar zou dan een breaking change introduceren hiervoor en dat leek me het niet direct waard.

Reply:
Ja, kan ik wel volgen. Als we het zouden willen veranderen naar een setter, is dit wel de moment natuurlijk. Maar niet per se nodig.

Reply:
Ik probeer altijd zo dicht mogelijk tegen de native syntax en conventies te blijven, daar wordt de callback als setter gedeclareerd. Bijvoorbeeld bij select: `onchange` setter. In dat opzicht zou ook hier een setter beter zijn. Ik twijfel wel of we daarvoor nu moeten breken.